packages/components/src/helpers/dummy.ts

```diff
@@ -177,300 +176,20 @@ export const dummyCardsColorful: CardsColorfulProps = {
-    },
-    {
-      title: 'GraphQL Modules 7',
-      description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
-      modal: {
-        header: {
-          image: {
-            src: marketplaceListImage,
-            alt: 'GraphQL Jit Logo',
-          },
-          description: {
-            href: '#',
-            children: 'Apr 20, 2021 - Latest Update',
-            title: 'Example Link Title',
-          },
-        },
-        content: 'Lipsum 7',
-      },
-      update: '2021-05-07T10:14:55.884Z',
-      image: {
-        src: marketplaceListImage,
-        alt: 'GraphQL Modules Logo',
-      },
-      link: {
-        href: '#',
-        title: 'Learn more about GraphQL Jit',
-      },
-    },
-    {
-      title: 'GraphQL Modules 8',
-      description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
-      modal: {
-        header: {
-          image: {
-            src: marketplaceListImage,
-            alt: 'GraphQL Jit Logo',
-          },
-          description: {
-            href: '#',
-            children: 'Apr 20, 2021 - Latest Update',
-            title: 'Example Link Title',
-          },
-        },
-        content: 'Lipsum 8',
-      },
-      update: '2021-05-07T10:14:55.884Z',
-      image: {
-        src: marketplaceListImage,
-        alt: 'GraphQL Modules Logo',
-      },
-      link: {
-        href: '#',
-        title: 'Learn more about GraphQL Jit',
-      },
-    },
-    {
-      title: 'GraphQL Modules 9',
-      description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
-      modal: {
-        header: {
-          image: {
-            src: marketplaceListImage,
-            alt: 'GraphQL Jit Logo',
-          },
-          description: {
-            href: '#',
-            children: 'Apr 20, 2021 - Latest Update',
-            title: 'Example Link Title',
-          },
-        },
-        content: 'Lipsum 9',
-      },
-      update: '2021-05-07T10:14:55.884Z',
-      image: {
-        src: marketplaceListImage,
-        alt: 'GraphQL Modules Logo',
-      },
-      link: {
-        href: '#',
-        title: 'Learn more about GraphQL Jit',
-      },
-    },
-    {
-      title: 'GraphQL Modules 10',
-      description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
-      modal: {
-        header: {
-          image: {
-            src: marketplaceListImage,
-            alt: 'GraphQL Jit Logo',
-          },
-          description: {
-            href: '#',
-            children: 'Apr 20, 2021 - Latest Update',
-            title: 'Example Link Title',
-          },
-        },
-        content: 'Lipsum 10',
-      },
-      update: '2021-05-07T10:14:55.884Z',
-      image: {
-        src: marketplaceListImage,
-        alt: 'GraphQL Modules Logo',
-      },
-      link: {
-        href: '#',
-        title: 'Learn more about GraphQL Jit',
-      },
-    },
-  ],
-};
-
 export const dummyMarketplaceSearch: IMarketplaceSearchProps = {
```

Comment:
let's move this file directly to story? it annoying files, I needed to configure `tsup` to manually ignore them from build in v8

Reply:
awesome, will do.
I'm personally not a fan of "one type" and directories like "dummy" or "types"